readme

```diff
@@ -106,3 +106,18 @@ CODE 200 OK
     }
 }
 
+----------------------------------------
+movieScreenings
+----------------------------------------
+GET /movie/:id/screenings
```

Comment:
I would have used plural in the first URL segment, i.e. `/movies/:id/screenings`, but the singular variation is also a common example.